README.md

```diff
@@ -1 +1,55 @@
-# terra-application
\ No newline at end of file
+<!-- Logo -->
+<p align="center">
+  <img height="128" width="128" src="https://github.com/cerner/terra-application/raw/master/terra.png" alt="terra logo" />
+</p>
+
+<!-- Name -->
+<h1 align="center">
+  Terra Application
+</h1>
+
+[![NPM version](https://badgen.net/npm/v/terra-application)](https://www.npmjs.org/package/terra-application)
+[![Cerner OSS](https://badgen.net/badge/Cerner/OSS/blue)](http://engineering.cerner.com/2014/01/cerner-and-open-source/)
+[![License](https://badgen.net/github/license/cerner/terra-application)](https://github.com/cerner/terra-application/blob/master/LICENSE)
+[![Build Status](https://badgen.net/travis/cerner/terra-application)](https://travis-ci.com/cerner/terra-application)
+[![Dependencies status](https://badgen.net/david/dep/cerner/terra-application)](https://david-dm.org/cerner/terra-application)
+[![devDependencies status](https://badgen.net/david/dev/cerner/terra-application)](https://david-dm.org/cerner/terra-application?type=dev)
+
+A framework to support application development with Terra components.
+
+* [Getting Started](#getting-started)
+* [Versioning](#versioning)
+* [Contributing](#contributing)
+* [LICENSE](#license)
+
+## Getting Started
+
+* Install with [npm](https://www.npmjs.com): `npm install --save terra-application`
+
+## Peer Dependencies
+
+terra-application requires no peer dependencies to be installed to properly function.
```

Comment:
[We should add the react peer dependencies](https://github.com/cerner/terra-application/pull/1/files?file-filters%5B%5D=.ico&file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.md&file-filters%5B%5D=.png&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap&file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R109)